johnsonandjohnson/Bodiless-JS
# feat(Compoenet): "Filterable Metadata" Component

## Changes
- Add metadata (tags/groups) to a component that allows an end user to hide it with a filter.

## Test Instructions
- Visit /filter-item on the test site.
- See AC's at #222  

## Related Issues
Closes #222
